main.go

```diff
@@ -283,7 +283,6 @@ func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
 	}
 	if err := (&remote.ClusterCacheReconciler{
 		Client:           mgr.GetClient(),
-		Log:              ctrl.Log.WithName("remote").WithName("ClusterCacheReconciler"),
```

Comment:
Q: does this will be replaced by the info being added with https://github.com/kubernetes-sigs/controller-runtime/pull/1827 in CR

Reply:
I'll move this to a new conversation